packages/api/src/webhooks/cannon.ts

```diff
@@ -603,6 +603,8 @@ export default class WebhookCannon {
         this.secondaryRecordObjectStoreId,
       );
 
+      const user = await db.user.get(session.userId);
```

Comment:
Maybe we should move this fetch https://github.com/livepeer/livepeer-com/blob/c87b3a1176de9ffca972c6ff6750149ec70a0659/packages/api/src/webhooks/cannon.ts#L173 to the beginning of the function and then pass the `user` around?

Otherwise if that's too big of a change for now, we can probably add `useCache` to both of those fetches (that and this) to hopefully avoid a db round trip